packages/x-date-pickers-pro/src/DesktopDateRangePicker/tests/DesktopDateRangePicker.test.tsx

```diff
@@ -220,7 +221,7 @@ describe('<DesktopDateRangePicker />', () => {
       const onChange = spy();
       const onAccept = spy();
       const onClose = spy();
-      const defaultValue: DateRange<any> = [
+      const defaultValue: DateRange<PickerValidDate> = [
```

Comment:
Couldn't you use `PickerRangeValue` here? 🤔

Reply:
It's not exported from the package, so I went for the public API type instead of using relating import to a private file.